app/assemblers/assemble_streaming_events.rb

```diff
@@ -0,0 +1,21 @@
+class AssembleStreamingEvents
+  include Mandate
+
+  initialize_with :params
+
+  def self.events_per_page = 20
+  def self.keys = %i[page per_page scheduled]
+
+  def call = SerializePaginatedCollection.(events, serializer: SerializeStreamingEvents)
+
+  private
+  memoize
+  def events
+    rows = scheduled? ? StreamingEvent.scheduled : StreamingEvent.live
+    rows.order(:starts_at).page(page).per(per_page)
+  end
+
+  def scheduled? = params[:scheduled].present?
```

Comment:
I'd be sort of tempted to invert this. I feel like `scheduled` should be the default. So I wonder if the param should be `live` instead?